src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs

```diff
@@ -118,3 +118,13 @@
    [:span {:style {:fontWeight 500}} namespace "/" name]
    [:span {:style {:fontWeight 200 :paddingLeft "1em"}} "Snapshot ID: "]
    [:span {:style {:fontWeight 500}} snapshot-id]])
+
+
+
+(defn render-name-id [name snapshot-id]
+  [:div {}
+   [:span {:style {:fontWeight 500}} name]
+   [:span {:style {:fontWeight 200 :paddingLeft "1em"}} "Snapshot ID: "]
+   [:span {:style {:fontWeight 500}} snapshot-id]])
```

Comment:
There's a lot of duplication between `render-entity` above, and `render-name-id`.

Could you refactor `render-entity` to concatenate its namepace and name arguments, and then call `render-name-id`?